src/main/resources/view/AppointmentListCard.fxml

```diff
@@ -19,15 +19,16 @@
       <padding>
         <Insets bottom="5" left="15" right="5" top="5" />
       </padding>
-      <HBox alignment="CENTER_LEFT" spacing="5">
-        <Label fx:id="id" styleClass="cell_big_label">
-          <minWidth>
-            <!-- Ensures that the label text is never truncated -->
-            <Region fx:constant="USE_PREF_SIZE" />
-          </minWidth>
-        </Label>
-        <Label fx:id="name" styleClass="cell_big_label" text="\$name" wrapText="true" />
-      </HBox>
+        <HBox alignment="CENTER_LEFT" spacing="5">
+          <Label fx:id="id" styleClass="cell_big_label">
+            <minWidth>
+              <!-- Ensures that the label text is never truncated -->
+              <Region fx:constant="USE_PREF_SIZE" />
+            </minWidth>
+          </Label>
+          <Label fx:id="name" styleClass="cell_big_label" text="\$name" wrapText="true" />
+          <Label fx:id="priority" alignment="CENTER" minWidth="90" styleClass="priority-text" wrapText="true" />
+        </HBox>
```

Comment:
Can you un-indent the entire block by 1 space so only the priority is a change.

Reply:
thx will reformate